### PR TITLE
fix regression tests

### PR DESCRIPTION
**Pull request by ofrimasad**

### CODE REVIEW

File: tests/recipe_training_tests/shortened_recipes_accuracy_test.py [hunk 1]
```diff
@@ -3,9 +3,10 @@
 import os
 import torch
 
-from super_gradients.common.environment.checkpoints_dir_utils import get_checkpoints_dir_path
+from super_gradients.common.environment.checkpoints_dir_utils import get_latest_run_dir_path
```
Comment (BloodAxe) [inline tests/recipe_training_tests/shortened_recipes_accuracy_test.py:6]: I cannot find get_latest_run_dir_path in master. This feature branch also don't have this function added. Am I missing something @ofrimasad ?

Comment (ofrimasad) [reply tests/recipe_training_tests/shortened_recipes_accuracy_test.py:6]: true.
moving this to draft mode until @Louis-Dupont fix the DDP <> run Id issue

**Comment by Louis-Dupont** [comment]
Note:
requires https://github.com/Deci-AI/super-gradients/pull/1415 to be merged first

**Review by BloodAxe — APPROVED** [review]
LGTM
